dydxV4/dydxV4/AppDelegate.swift

```diff
@@ -143,15 +144,7 @@ class AppDelegate: CommonAppDelegate {
     }
 
     override func deepLinkHandled(deeplink: URL, successful: Bool) {
-        let data = ["url": deeplink.absoluteString]
-        if successful {
-            if let type = deeplink.params?["notification_type"] {
-                Tracking.shared?.view("/notification/deeplink/" + type, data: data)
-            }
-            Tracking.shared?.view("/notification/deeplink/success", data: data)
-        } else {
-            Tracking.shared?.view("/notification/deeplink/failure", data: data)
-        }
```

Comment:
condensed into a single event with an event name instead of a path. Lmk if u disagree with this decision